Unittest_camera_distance.py

```diff
@@ -0,0 +1,70 @@
+import unittest
+import numpy as np
+from enum import Enum
+from vision.common.constants import Point, CameraParameters, ODLCDict
+from vision.common.bounding_box import BoundingBox, ObjectType
+from vision.deskew.camera_distances import get_coordinates, bounding_area, calculate_distance
+
+
+class TestVisionFunctions(unittest.TestCase):
+
+    def setUp(self) -> None:
+        # Set up the camera parameters and other constants for each test to avoid cross-contamination
+        self.camera_params = CameraParameters(
+            focal_length=35.0,
+            rotation_deg=[0, 0, 0],
+            drone_coordinates=[37.7749, -122.4194],
+            altitude_f=1000.0,
+        )
+        self.image_shape = (1080, 1920, 3)  # Image size with 3 color channels
+
+    def test_get_coordinates(self) -> None:
+        # Test the coordinates calculation at the center of the image
+        center_pixel = (960, 540)
+        expected_coordinates = (37.77, -122.211)
+        result = get_coordinates(center_pixel, self.image_shape, self.camera_params)
+
+        self.assertIsNotNone(result, "Coordinates should not be None")
```

Comment:
After checking that a result is not None, it would also be a good idea to explicitly check that the type exactly matches the expected format.

Reply:
Please do this for every test function, for brevity I will not add a comment for each instance.